core/src/test/java/org/mskcc/oncotree/utils/VersionUtilTest.java

```diff
@@ -20,83 +20,33 @@
 import java.io.*;
 import java.nio.file.*;
 import java.util.*;
-import org.junit.AfterClass;
+import javax.annotation.Resource;
 import org.junit.Before;
-import org.junit.BeforeClass;
 import org.junit.runner.RunWith;
 import org.junit.Test;
-import org.mockito.InjectMocks;
-import org.mockito.Mock;
-import org.mockito.Mockito;
-import org.mockito.MockitoAnnotations;
 import org.mskcc.oncotree.error.InvalidVersionException;
-import org.mskcc.oncotree.model.MainType;
 import org.mskcc.oncotree.model.Version;
-import org.mskcc.oncotree.topbraid.OncoTreeNode;
-import org.mskcc.oncotree.topbraid.OncoTreeVersionRepository;
 import org.springframework.beans.factory.annotation.Autowired;
-import org.springframework.context.annotation.AnnotationConfigApplicationContext;
-import org.springframework.context.annotation.Configuration;
 import org.springframework.context.annotation.Import;
-import org.springframework.context.support.ClassPathXmlApplicationContext;
-import org.springframework.test.context.ContextConfiguration;
 import org.springframework.test.context.junit4.SpringRunner;
-import org.springframework.test.context.web.WebAppConfiguration;
-import static org.junit.Assert.assertEquals;
-import static org.junit.Assert.assertFalse;
-import static org.junit.Assert.assertNotNull;
-import static org.junit.Assert.assertTrue;
 import static org.junit.Assert.fail;
 import static org.mockito.Matchers.*;
-import static org.mockito.Mockito.*;
-
 
 @RunWith(SpringRunner.class)
-@Import(VersionUtilTestConfig.class)
+@Import(OncotreeTestConfig.class)
 public class VersionUtilTest {
-    private List<Version> oncoTreeVersionRepositoryMockResponse = null;
-    @Autowired
-    private OncoTreeVersionRepository mockVersionRepository = null;
+    @Resource(name="oncoTreeVersionRepositoryMockResponse")
+    private List<Version> oncoTreeVersionRepositoryMockResponse;
+
     @Autowired
-    private VersionUtil versionUtil = null;
-    //TODO convert this class to use the common OncotreeConfiguration class for unit tests. Proviede the expectedVersionMap as a @Resource
+    private VersionUtil versionUtil;
+
+    //TODO convert this class to use the common OncotreeConfiguration class for unit tests. Provide the expectedVersionMap as a @Resource
```

Comment:
Part of this TODO is now done ... the "common OncotreeConfiguration class" is now used, so that part can be deleted.